lib/zipkin-tracer.rb

```diff
@@ -17,6 +17,15 @@
 
 require 'zipkin-tracer/careless_scribe'
 
+if RUBY_PLATFORM == 'java'
+  begin
+    require 'hermann/producer'
+  rescue LoadError
+    # Hermann not available
+  end
+  require 'zipkin-tracer/zipkin_kafka_tracer' if defined?(::Hermann)
```

Comment:
If Hermann cannot be loaded at this time, will the tracer actually operate correctly? I would expect this to be a failure instead of a recoverable error